doc/man1/openssl.pod

```diff
@@ -554,6 +562,73 @@ Read the password from standard input.
 
 =back
 
+=head2 Trusted Certificate Options
+
+Part of validating a certificate includes verifying that the chain of CA's
+can be tied up to an existing trusted root.  The following options specify
+how to list the trusted roots, also known as trust anchors.  A collection
+of trusted roots is called a I<trust store>.
+
+Note that OpenSSL does not provider a default set of trust anchors.  Many
```

Comment:
We're all so used to say "provider" these days, aren't we?  I constantly catch myself making that exact same mistake

Reply:
Ha!  Fixed.